include/communication/service/communication_service.h

```diff
@@ -0,0 +1,44 @@
+#ifndef COMMUNICATION_SERVICE_H
+#define COMMUNICATION_SERVICE_H
+
+#include "i_service.h"
```

Comment:
move to internal headers

Reply:
Do u mean to cpp? But we inherit from IService and we need this in the header

Reply:
No, just place include with other internal includes

Reply:
I thought that we place parent class headlers first, as in here - https://github.com/Midgrad/Dreka/wiki/Coding-Styles 

But ok